packages/babel-traverse/src/scope/index.js

```diff
@@ -140,7 +140,7 @@ const collectorVisitor = {
   },
 
   UpdateExpression(path, state) {
-    state.constantViolations.push(path.get("argument"));
+    state.constantViolations.push(path);
   },
 
   UnaryExpression(path, state) {
```

Comment:
I removed `UnaryExpression` from the visitor and it did not affect any tests. Should I delete it?

I tried rebasing locally by running `git pull --rebase upstream 7.0` and it did fix the broken test. However it says I have a diverging history from my fork and wants me to do another merge commit by pulling from my fork/consistent-violations. Does that sound right or is there a better way?